test/testnet/localnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Remote Version of Localnet
+
+Remote version of the test network generated by running:
+
+```bash
+> make localnet-start
+```
+
+from the Tendermint repository. There are a few additional minor tweaks, however:
+
+1. The proxy app is preconfigured to `kvstore`
+2. Logging format is JSON
+3. `persistent_peers` is preconfigured
+4. `create_empty_blocks` is set to `false`
+5. `prometheus` is set to `true` to allow for collecting Prometheus metrics
+
+## Contents
+This folder contains the relevant prerequisites to deploy this test network to a
+set of predefined hosts. Here is a description of a subset of the layout of this
+folder's contents:
+
+```
+|_ Makefile           The primary entrypoint to interacting with the test network.
+|_ requirements.txt   Python requirements for running the deployment scripts.
+|_ config             Ansible configuration management.
+   |_ hosts           The inventory file for the hosts to which we will be deploying the network.
+   |_ (secrets.yml)   NOT PROVIDED. See the "Secrets" section below for details.
+|_ nodes              Tendermint config/data for each node in the `config/hosts` file.
+```
+
+## Prerequisites
+To be able to deploy this test network, you will need the following software
+installed:
+
+* make
+* Python 3
+* Go 1.11.4+
+* Tendermint source under `${GOPATH}/src/github.com/tendermint/tendermint`
+
+## Secrets
+In order to deploy this network, you will first need to set up a `secrets.yml`
+file inside the `config` folder with the following structure:
+
+```yaml
+secrets:
+  nodes:
+    node0:
+      address: "<redacted>"      # Validator address for this node
+      public_key: "<redacted>"
+      private_key: "<redacted>"
+      node_key: "<redacted>"     # Node private key (ed25519)
+      power: "1"
+    node1:
+      address: "<redacted>"
+      public_key: "<redacted>"
+      private_key: "<redacted>"
+      node_key: "<redacted>"
+      power: "1"
+    # etc., same for node2 and node3
+```
```

Comment:
In AWS, this can actually be polled from an API. (At least the network parts). So, as I said, maybe it's worth rethinking this.